packages/min-100-layout/index.test.js

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from '@testing-library/react'
+import Min100Layout from '.'
+
+describe('<Min100Layout />', () => {
+  it('should pass a provided className to the root element', () => {
```

Comment:
This is a pretty minimal render-some-CSS component, so these are pretty minimal tests.